From 93865ebbb08286798681375a34a6855a3c19fa18 Mon Sep 17 00:00:00 2001
From: Yunlei Ding <yunlei.ding@citrix.com>
Date: Fri, 18 Apr 2014 06:23:24 +0100
Subject: virtio-blk: initialize unused blkcfg.size_max field
To: david.vrabel@citrix.com

Coverity complains that this is not initialized (even though it is
never used).

Signed-off-by: Yunlei Ding <yunlei.ding@citrix.com>
diff --git a/hw/virtio-blk.c b/hw/virtio-blk.c
index 520ad1b..fcf893a 100644
--- a/hw/virtio-blk.c
+++ b/hw/virtio-blk.c
@@ -252,6 +252,7 @@ static void virtio_blk_update_config(VirtIODevice *vdev, uint8_t *config)
     stw_raw(&blkcfg.cylinders, cylinders);
     blkcfg.heads = heads;
     blkcfg.sectors = secs;
+    blkcfg.size_max = 0;
     memcpy(config, &blkcfg, sizeof(blkcfg));
 }
 
